templates/zerver/register.html

```diff
@@ -102,7 +102,20 @@ <h1>{{ _('Create your account') }}</h1>
                     <input type='hidden' name='key' value='{{ key }}' />
                     <input type='hidden' name='timezone' id='timezone'/>
                     <label for="id_email" class="inline-block label-title">{{ _('Email') }}</label>
-                    <div id="id_email">{{ email }}</div>
+                    <div class="email-visibility-text">
+                        {% if EMAIL_ADDRESS_VISIBILITY == "1" %}
+                        All the admins, members and guests will be able to see this email address.
```

Comment:
This text is too long and the is rendered in multiple lines which does not look good. Maybe we can use "Everyone will be able to see this email address" here as well.
Also we do not need to pass different value "0" for new realm, we can simply use Realm.EMAIL_ADDRESS_VISIBILITY_EVERYONE as it is the default value for this setting.
I would also like to know Tim's thoughts on this.

Reply:
Okay sure.I will do it to the default settings instead of 0.